memento-notes/src/pages/Auth/Login/Login.jsx

```diff
@@ -0,0 +1,56 @@
+import React,{useState} from 'react'
+import {Header} from '../../../components'
+import './Login.css'
+import { useNavigate } from "react-router-dom";
+import {useAuth} from '../../../hooks'
+import {LoginService} from '../../../services'
+
+export const Login = () => {
+    const navigate = useNavigate();
+    const {auth,setAuth} = useAuth();
+    const [login,setLogin] = useState({
+        email:'',
+        password:'',
+    })
+
+    const inputHandler =(e)=>{
+        setLogin((prev)=>({...prev,[e.target.name]:e.target.value}))
```

Comment:
handle e.target.name paramenter validation. undefined value of name might give error, because you are using as object property.

Reply:
okay,will validate the form